manifests/o/Obsidian/Obsidian/1.5.3/Obsidian.Obsidian.installer.yaml

```
@@ -0,0 +1,31 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Obsidian.Obsidian
+PackageVersion: 1.5.3
+InstallerType: nullsoft
+InstallModes:
+- silent
+UpgradeBehavior: install
+FileExtensions:
+- md
+ReleaseDate: 2023-12-26
+Installers:
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://github.com/obsidianmd/obsidian-releases/releases/download/v1.5.3/Obsidian.1.5.3.exe
+  InstallerSha256: BFBB6BE48E545AF81D6EA0C66FF8F41D19FCD1AFC21499D44FA89C9F1241BF42
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://github.com/obsidianmd/obsidian-releases/releases/download/v1.5.3/Obsidian.1.5.3-32.exe
+  InstallerSha256: BFBB6BE48E545AF81D6EA0C66FF8F41D19FCD1AFC21499D44FA89C9F1241BF42

```

Hi @BiosPlus, the architecture for this one is `x86`.

```suggestion
- Architecture: x86
  Scope: user
  InstallerUrl: https://github.com/obsidianmd/obsidian-releases/releases/download/v1.5.3/Obsidian.1.5.3-32.exe
  InstallerSha256: BFBB6BE48E545AF81D6EA0C66FF8F41D19FCD1AFC21499D44FA89C9F1241BF42
```

---

Cheers @rodrigoborgesdeoliveira